### ISSUE TITLE
Uncaught (in promise) Error: Assertion failed. --> Naja.js:1398:2

### ISSUE THREAD

**Issue by altosch** [jiripudil/Naja#197]
# Bug Report

The form in the modal window. When [SEND] button is clicked the modal window disappears but the backdrop remains and the browser tab is unresponsive until refresh. Values from the form are sent as expected and the snippet is updated properly.

The console shows:

* Firefox:
Uncaught (in promise) Error: Assertion failed. --> Naja.js:1398:2
    AssertionError Naja.js:1398
    assert utils.ts:5
    clickElement UIHandler.ts:95
    handleUI UIHandler.ts:75
    bindElement UIHandler.ts:35
    bindUI UIHandler.ts:40
    initialize UIHandler.ts:18
    updateSnippet SnippetHandler.ts:66
    updateSnippets SnippetHandler.ts:29
    updateSnippets SnippetHandler.ts:26
    SnippetHandler SnippetHandler.ts:20
    makeRequest Naja.ts:154
    clickElement UIHandler.ts:118
    handleUI UIHandler.ts:75
    bindElement UIHandler.ts:35
    bindUI UIHandler.ts:40
    initialize UIHandler.ts:15
    initialize Naja.ts:73
    <anonymous> (index):32
    EventListener.handleEvent* (index):32
    clickElement UIHandler.ts:119
    handleUI UIHandler.ts:75
    handleUI self-hosted:1161
    (Async: EventListener.handleEvent)
    bindElement UIHandler.ts:35
    bindUI UIHandler.ts:40
    initialize UIHandler.ts:18
    updateSnippet SnippetHandler.ts:66
    updateSnippets SnippetHandler.ts:29
    forEach self-hosted:206
    updateSnippets SnippetHandler.ts:26
    SnippetHandler SnippetHandler.ts:20
    makeRequest Naja.ts:154
    AsyncFunctionNext self-hosted:674
    (Async: async)
    clickElement UIHandler.ts:118
    handleUI UIHandler.ts:75
    handleUI self-hosted:1161
    (Async: EventListener.handleEvent)
    bindElement UIHandler.ts:35
    bindUI UIHandler.ts:40
    initialize UIHandler.ts:15
    initialize self-hosted:1161
    initialize Naja.ts:73
    <anonymous> (index):32
    (Async: EventListener.handleEvent)
    <anonymous> (index):32

* Chrome
polyfills.js:10 Uncaught (in promise) Error: Assertion failed.
    at assert (utils.ts:5)
    at EventTarget.clickElement (UIHandler.ts:95)
    at EventTarget.handleUI (UIHandler.ts:75)
AssertionError @ polyfills.js:10
assert @ utils.ts:5
clickElement @ UIHandler.ts:95
handleUI @ UIHandler.ts:75

## Code that reproduces the problem
```
{snippet main}
<div class="modal fade" id="myformModal" tabindex="-1" role="dialog">
    <div class="modal-dialog modal-lg" role="document">
	<form n:name=dokumentyForm class="ajax">
	    <div class="modal-content">		
		<div class="modal-body">
		    <div class="form-group row">
			<label n:name="item" class="col-sm-2 col-form-label">Item</label>
			<div class="col-sm-10">                                
			    <input n:name="item" class="form-control form-control-sm mr-1" style="font-weight: bold;">
			</div>
		    </div>
		</div>
		<div class="modal-footer">
		    <button n:name="send" type="submit" class="btn btn-primary float-right">SEND</button>                             
		</div> 
	    </div>
	</form>
    </div>
</div>

<script>
    $(document).ready(function () {
	$('#myformModal').modal('show');
    });
</script>

<div class="row">
    <div class="col-12">        	
            <table class="table table-bordered table-hover mt-2">
                           {* data *}
            </table> 
    </div>
</div>

{/snippet}
```

Versions:
Naja.js 2.1.1
Bootstrap v4.5.3
nette/application  v3.1.0
Firefox 84.0.2, Chrome 87.0.4280

**Comment by jiripudil** [comment]
Yep, that's a bug, thanks for the report. If you need a quick remedy before this is fixed, you can replace the `button` with an `input` element.

**Comment by jiripudil** [comment]
Fixed in [2.1.2](https://github.com/jiripudil/Naja/releases/tag/2.1.2)
